File: Xpand.DesignExperience/vs_templates/cs/_PROJECTNAME_.Module/Module.cs
using DevExpress.ExpressApp.Security.Strategy;
using DevExpress.Persistent.BaseImpl;
using DevExpress.ExpressApp;
using System.Reflection;
using Xpand.ExpressApp.JobScheduler.Jobs.ThresholdCalculation;
using Xpand.ExpressApp.ModelDifference.Security;

namespace $projectsuffix$.Module {
    public sealed partial class $projectsuffix$Module : ModuleBase {
        public $projectsuffix$Module() {
            AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(typeof(Analysis)), IsExportedType));
            AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(typeof(Xpand.Persistent.BaseImpl.Updater)), IsExportedType));
            AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(typeof(ThresholdSeverity)), IsExportedType));
            InitializeComponent();
        }
    }
}

File: Xpand.DesignExperience/vs_templates/vb/_PROJECTNAME_.Module/Module.vb

Imports DevExpress.ExpressApp
Imports System.Reflection
Imports DevExpress.Persistent.BaseImpl
Imports Xpand.ExpressApp.JobScheduler.Jobs.ThresholdCalculation

Partial Public NotInheritable Class [$projectsuffix$Module]
    Inherits ModuleBase
    Public Sub New()
        AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(GetType(Analysis)), AddressOf IsExportedType))
        AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(GetType(Xpand.Persistent.BaseImpl.Updater)), AddressOf IsExportedType))
        AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(GetType(ThresholdSeverity)), AddressOf IsExportedType))
        InitializeComponent()
    End Sub

End Class
